src/test/java/LoginTests.java

```diff
@@ -1,26 +1,21 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
-import java.time.Duration;
-
-public class LoginTests extends BaseTest {
-    @Test
-    public void loginEmptyEmailPassword() {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
 
-        String url = "https://qa.koel.app/";
+public class LoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage = new LoginPage(driver);
         driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
-       //Luda
+    }
+    @Test
+    public void validLogin(){
+        loginPage.loginCorrectCred();
+    }
+    @Test
+    public void validLoginOnceLoggedIn(){
+        loginPage.loginCorrectCred();
```

Comment:
Please, add Assert

**One test - One Assert**

![Clean_Test](https://github.com/LiudmilaChernyh/A49-QA-Automation-Framework/assets/104761482/da64e15e-0340-4dff-abad-32da849dbea4)

Reply:
When I add assertion the whole code is failing. How to add it in invalidTest? if I don't have Home Page but only Login and Base Pages. To what to assert to Login Page or to Base Page?

Reply:
> When I add assertion the whole code is failing. How to add it in invalidTest? if I don't have Home Page but only Login and Base Pages. To what to assert to Login Page or to Base Page?

For example

_Download_

1) Code with Russian comments https://drive.google.com/file/d/1vD6sCpCuJbBaa3JexJQJPMbL7LHQKiIG/view?usp=sharing


2) Code with English comments 
https://drive.google.com/file/d/1UT177UX39eWqt9BkxTgDFrsqG5AZ_OLU/view?usp=sharing

Reply:
ok. But I am still confused where to put assertion, to pages or to tests?